configmap/mariadb/start/initdb.sh

```diff
@@ -9,7 +9,8 @@
 # @author  Fabrice Jammes, IN2P3/SLAC
 
 set -e
-set -x
+# WARN: password are displayed in debug logs
```

Comment:
For the record. This is not the case for the Replication system which is supposed to replace passwords with some hash like `xxxxxxx` or `*******`. Please, let me know if tis is  not the case. Then I should fix the code.

Reply:
Password are displayed in initdb.log because of the `set -x. I think Replication system does not display them.